romancal/regtest/regtestdata.py

```diff
@@ -383,20 +385,20 @@ def is_like_truth(rtdata, ignore_asdf_paths, output, truth_path, is_suffix=True)
     """
     __tracebackhide__ = True
     # If given only a suffix, get the root to change the suffix of.
-    # If the input was an association, the output should be the name of the product
-    # Otherwise, output is based on input.
+    # If the input was an association, the output should be the name of
+    # the product. Otherwise, output is based on input.
     if is_suffix:
-        #suffix = output
+        # suffix = output
         if rtdata.asn:
             output = rtdata.asn['products'][0]['name']
         else:
             output = os.path.splitext(os.path.basename(rtdata.input))[0]
-        #output = replace_suffix(output, suffix) + '.asdf'
+        # output = replace_suffix(output, suffix) + '.asdf'
     rtdata.output = output
 
     rtdata.get_truth(os.path.join(truth_path, output))
 
-    #diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
+    # diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
```

Comment:
Hm, line 401 is a comment. I wonder why codecov is reporting it as not covered.

Reply:
That is a good question and I cannot find a good answer. It appears that the space after the # may be confusing codecov since "# command" can be used to pass commands to codecov. I'd have to do some testing to be sure.